lib/node_modules/@stdlib/lapack/base/dtrti2/test/test.dtrti2.js

```diff
@@ -0,0 +1,261 @@
+
+
+// FUNCTIONS //
+
+/**
+* Tests for element-wise approximate equality.
+*
+* @private
+* @param {Object} t - test object
+* @param {Collection} actual - actual values
+* @param {Collection} expected - expected values
+* @param {number} rtol - relative tolerance
+*/
+function isApprox( t, actual, expected, rtol ) {
+	var delta;
+	var tol;
+	var i;
+
+	t.strictEqual( actual.length, expected.length, 'returns expected value' );
+	for ( i = 0; i < expected.length; i++ ) {
+		if ( actual[ i ] === expected[ i ] ) {
+			t.strictEqual( actual[ i ], expected[ i ], 'returns expected value' );
+		} else {
+			delta = abs( actual[ i ] - expected[ i ] );
+			tol = rtol * EPS * abs( expected[ i ] );
+			t.ok( delta <= tol, 'within tolerance. actual: '+actual[ i ]+'. expected: '+expected[ i ]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+}
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof dtrti2, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 6', function test( t ) {
+	t.strictEqual( dtrti2.length, 6, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function throws an error if provided an invalid first argument', function test( t ) {
+	var values;
+	var A;
+	var i;
+
+	values = [
+		'foo',
+		'bar',
+		'beep',
+		'boop',
+		5,
+		true,
+		false
+	];
+
+	A = new Float64Array( [ 1.0, 2.0, 0.0, 4.0 ] );
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[ i ] ), TypeError, 'throws an error when provided ' + values[ i ] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			dtrti2( value, 'upper', 'non-unit', 2, A, 2 );
+		};
+	}
+});
+
+tape( 'the function throws an error if provided an invalid second argument', function test( t ) {
+	var values;
+	var A;
+	var i;
+
+	values = [
+		'foo',
+		'bar',
+		'beep',
+		'boop',
+		5,
+		true,
+		false
+	];
+
+	A = new Float64Array( [ 1.0, 2.0, 0.0, 4.0 ] );
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[ i ] ), TypeError, 'throws an error when provided ' + values[ i ] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			dtrti2( 'row-major', value, 'non-unit', 2, A, 2 );
+		};
+	}
+});
+
+tape( 'the function throws an error if provided an invalid third argument', function test( t ) {
+	var values;
+	var A;
+	var i;
+
+	values = [
+		'foo',
+		'bar',
+		'beep',
+		'boop',
+		5,
+		true,
+		false
+	];
+
+	A = new Float64Array( [ 1.0, 2.0, 0.0, 4.0 ] );
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[ i ] ), TypeError, 'throws an error when provided ' + values[ i ] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			dtrti2( 'row-major', 'upper', value, 2, A, 2 );
+		};
+	}
+});
+
+tape( 'the function correctly computes inverse of upper triangular non-unit matrix ( row-major )', function test( t ) {
```

Comment:
```suggestion
tape( 'the function correctly computes inverse of upper triangular non-unit matrix (row-major)', function test( t ) {
```

Remove spaces here and in other test descriptions.